src/init/init.ts

```diff
@@ -174,7 +170,7 @@ export function runGenerator(generatorName, options): Promise<any> {
 
   return new Promise((resolve, reject) => {
```

Comment:
this could just be: `const env = await options.env || createYeomanEnvironment();`